packages/app/src/app/pages/Dashboard/Content/SandboxCard/index.tsx

```diff
@@ -81,6 +81,18 @@ type State = {
 export const DELETE_SANDBOX_DROP_KEY = 'delete';
 export const MAKE_TEMPLATE_DROP_KEY = 'makeTemplate';
 
+const copyToClipboard = (str: string) => {
```

Comment:
This is awesome! But we already have this function here: 
https://github.com/codesandbox/codesandbox-client/blob/63ea50dd95384eb2bff27b33e29fce4b139020ea/packages/app/src/app/overmind/effects/browser.ts#L54

You can call it from Overmind with `effects.browser.copyToClipboard` and passing the string 

Let me know if you need any help

Reply:
Hey 👋 

Thanks for the quick review, will make this change in some time.

Reply:
Cool! Thank you!

Reply:
As we have not refactored the Component yet so is there a way to access `effects` in Class Components? I was thinking to access the util by `useOvermind` hook but this won't work here.

Reply:
You are very right!

I will merge it!

Thank you!